modules/floating-buttons/module.php

```diff
@@ -105,6 +105,8 @@ public function __construct() {
 
 		add_action( 'save_post_' . static::CPT_FLOATING_BUTTONS, [ $this, 'flush_permalinks_on_save' ] );
 
+		add_action( 'elementor/frontend/after_register_styles', [ $this, 'register_style' ] );
```

Comment:
The enqueue order can cause issues, because we now load the inline widget CSS before the general widget CSS.

If this works for you, we can merge it. I might update the enqueue action hook next week, to ensure that we implement this in a consistent way with the other widgets.

We could also wait a couple of days, until this task is merged: https://elementor.atlassian.net/browse/ED-15011.

Wdyt, @louiswol94 ?

Reply:
@hein-obox Thanks for the review!

I asked Aviv that we hold off on this PR till Rami's is merged so that we fully know which approach to follow :)